components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -448,7 +455,7 @@ - (void)updateTrailingViewAlpha {
 - (CGFloat)trailingViewAlpha {
   // The trailing view has the same behavior as .rightView in UITextField: It has visual precedence
   // over the clear button.
-  CGFloat trailingViewAlpha = self.trailingView.alpha;
```

Comment:
Also, I had that line as a subliminal reminder to the next dev what we're talking about: self.trailingView's alpha.